File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_api_result_plus_organization_member_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusApiResultPlusOrganizationMemberVO:
    """API调用结果"""
    data: PlusOrganizationMemberVO
    code: str
    msg: str
    request_id: str
    ip: str = None
    hostname: str = None
    error_name: str

File: sdkwork-backend-sdk-go/types/plus_api_result_plus_organization_member_vo.go
package types

// API调用结果
type PlusApiResultPlusOrganizationMemberVO struct {
	Data PlusOrganizationMemberVO `json:"data"`
	Code string `json:"code"`
	Msg string `json:"msg"`
	RequestId string `json:"requestId"`
	Ip string `json:"ip"`
	Hostname string `json:"hostname"`
	ErrorName string `json:"errorName"`
}
